app/lib/notifications.server.ts

```diff
@@ -0,0 +1,35 @@
+import { createTransport, type SendMailOptions } from 'nodemailer';
+import { secrets } from 'docker-secret';
+import logger from './logger.server';
+
+const { NODEMAILER_USER } = process.env;
+const { NODEMAILER_CLIENT_ID, NODEMAILER_ACCESS_TOKEN } = secrets ?? {};
+
+const initializeTransport = () => {
+  if (!(process.env.NODEMAILER_USER && NODEMAILER_CLIENT_ID && NODEMAILER_ACCESS_TOKEN)) {
+    throw new Error('Missing Nodemailer environment variables, skipping nodemailer configuration');
+  }
+  const transporter = createTransport({
+    service: 'Outlook365',
+    auth: {
+      type: 'OAuth2',
+      user: NODEMAILER_USER, // email address for notifications
+      clientId: NODEMAILER_CLIENT_ID,
+      accessToken: NODEMAILER_ACCESS_TOKEN,
+      expires: 1484314697598,
```

Comment:
What time format is this? Epoch? Seems like epoch translates into `Thu, 23 Jan 49006 01:13:18`.

Reply:
This is the "default" expiration time in milliseconds. I should give it a more realistic value. `expires` accepts value in ms.

Reply:
ahh okay make sense